sdks/js/packages/sdk-demo/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@raystack/apsara": "^0.16.1",
-    "@raystack/frontier": "^0.22.0",
+    "@raystack/frontier": "workspace:*",
```

Comment:
?

Reply:
This is the demo app for local testing. i use local reference of the SDK.